api/control.yaml

```diff
@@ -206,6 +204,43 @@ components:
         - id
         - name
         - email
+    UserResourceList:
+      description: List of SCIM User resources.
+      type: object
+      properties:
+        totalResults:
+          description: The total number of results.
+          type: integer
+          format: int32
+        itemsPerPage:
+            description: The number of items per page.
+            type: integer
+            format: int32
+        startIndex:
+            description: The index of the first result.
+            type: integer
+            format: int32
+        Resources:
+            description: The list of User resources.
+            type: array
+            items:
+                $ref: '#/components/schemas/UserResource'
+        id:
+            description: User list id metadata.
+            type: string
+        externalId:
+            description: User list external id metadata.
+            type: string
+        meta:
```

Comment:
why do we we want to have a meta on the user list? Is it to prevent errors from okta as well?

Reply:
Yes. oddly its part of the SCIM spec (or at least the SDK expects it)